pkgs/by-name/wr/wrkflw/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  docker,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wrkflw";
+  version = "unstable-2024-01-19";
```

see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning. the commit and pr title should also be updated
```suggestion
  version = "0-unstable-2024-01-19";
```